stbt/__init__.py

```diff
@@ -408,7 +408,9 @@ def ocr(frame=None, region=Region.ALL,
     :param text_color:
         Color of the text. Specifying this can improve OCR results when
         tesseract's default thresholding algorithm doesn't detect the text,
-        for example for white text on a light-colored background.
+        for example for white text on a light-colored background. The threshold
+        can be adjusted further by passing a 4th number in the tuple, also
+        between 0 and 255.
```

Comment:
I'm not sure of this as an API.  Perhaps a `text_color_threshold` would be better.  I've been thinking of adding a `Color` class much like we have `Region` and I don't see how it would fit in here.